code/notificationbuilder/src/main/res/layout/push_template_horizontal_catalog.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/catalog_container_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:clipToPadding="false"
+    android:clipChildren="false"
+    android:orientation="horizontal"
+    android:theme="@style/DayNightTheme">
+
+    <ImageView
+        android:id="@+id/product_image"
+        android:scaleType="fitXY"
+        android:adjustViewBounds="true"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/product_catalog_image_height"
+        android:layout_alignParentLeft="true"
+        android:contentDescription="@string/big_product_image">
+    </ImageView>
+
+    <LinearLayout
+        android:id="@+id/horizontal_catalog_mid_row_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/product_catalog_horizontal_mid_row_margin_top"
+        android:layout_below="@id/product_image">
+
+        <TextView
+            android:id="@+id/product_title"
+            style="@style/Title"
+            android:layout_weight="1"
+            android:layout_gravity="start"
+            android:layout_width="@dimen/product_catalog_horizontal_mid_row_width"
```

Comment:
Same comment here. Please set `layout_width` to 0 and use weights to control how much percentage each view must take